src/parser.py

```diff
@@ -239,6 +241,9 @@ def parse_file(
 
     actors_key: str = "Actors"
     data[actors_key] = {}
+    data[actors_key]["Id"] = []
```

Comment:
This is a fine change, but as is you'll need to fix the `example.py` script (Line ~470) which still uses the `"Id"` method to index into a dictionary containing `"Location"` and `"Rotation"`. Can you please fix that?